pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -1092,7 +1116,7 @@ func (c *K8sOrchestrator) IsFSSEnabled(ctx context.Context, featureName string)
 		c.supervisorFSS.featureStatesLock.RLock()
 		if flag, ok := c.supervisorFSS.featureStates[featureName]; ok {
```

Comment:
do we need similar change as supervisor cluster here for guest cluster as well to refer WCP feature state in GC ?? or will that be added whenever needed in future ??

Reply:
Implementing this functionality for GC will need more design discussions. For now, we are targeting only WCP.